### ISSUE TITLE
can't add hover style to .mm-counter digits

### ISSUE THREAD

**Issue by zuhloobie** [BeSite/jQuery.mmenu#396]
I can style the counter digits by targeting `.mm-counter {color:#333 !important}` in the CSS, but I can't for the life of me add a hover style. I want to be able to show a different color when you hover over the digits in the counter.

I've tried styling in CSS:
`.mm-counter:hover {color:#F00 !important};`
`em.mm-counter:hover {color:#F00 !important;}`
`.mm-menu ul li em:hover {color:#F00 !important;}`
...and others that I can't remember at the moment.

I've tried styling it via jQuery like this:
`	$('.mm-counter').hover(function(){
		$(this).css('color', '#F00');
	});
`

I've also tried adding a class to `.mm-counter` via jQuery and applying a hover to it that way. While it does add the class...it ignores the hover style.

Basically, I can style the heck out of the `<em class="mm-counter"></em>` EXCEPT for hover.

How can I achieve this?

Thanks!

**Comment by FrDH** [comment]
This is because the `.mm-counter` has a lower z-index than the `<a>` (or `<span>`). Try adding this CSS:
````css
.mm-counter {
   z-index: 3;
}
````
Should do the trick.

**Comment by zuhloobie** [comment]
Ugh - stumbled by something so simple. Perfect - thank you.

**Comment by zuhloobie** [comment]
Hmm - a side effect to increasing the z-index is that clicking on the counter digits no longer advances into the sub-menu :/ Now you can only see sub-menu by clicking the ::after icon ">"

**Comment by FrDH** [comment]
I don't think you can have both. I'm going to close this issue as it is not an issue with the mmenu plugin but more a general HTML/CSS question.
